### DISCUSSION TITLE
find: File system loop detected

### DISCUSSION THREAD

**Discussion by brubrant** [EA31337/EA-Tester#232] [Q&A]
Hi, I'm trying to use the tester (Manjaro Linux) but I'm getting the following erros:

find: File system loop detected; '/home/ubuntu/.wine/drive_c/users/ubuntu/Downloads' is part of the same file system loop as '/home/ubuntu'.
find: File system loop detected; '/home/ubuntu/.wine/drive_c/users/ubuntu/Templates' is part of the same file system loop as '/home/ubuntu'.
find: File system loop detected; '/home/ubuntu/.wine/drive_c/users/ubuntu/My Music' is part of the same file system loop as '/home/ubuntu'.
find: File system loop detected; '/home/ubuntu/.wine/drive_c/users/ubuntu/My Pictures' is part of the same file system loop as '/home/ubuntu'.
find: File system loop detected; '/home/ubuntu/.wine/drive_c/users/ubuntu/My Videos' is part of the same file system loop as '/home/ubuntu'.
find: File system loop detected; '/home/ubuntu/.wine/drive_c/users/ubuntu/My Documents' is part of the same file system loop as '/home/ubuntu'.
find: File system loop detected; '/home/ubuntu/.wine/dosdevices/z:/home/ubuntu' is part of the same file system loop as '/home/ubuntu'.

To be honest I'm fairly new to docker.

**Comment by kenorb** [comment]
Are you mounting local filesystem with `-v`? Can you try version from `dev` branch, it got some fixes for it I think (use `ea31337/ea-tester:dev`). Potentially 1f1b2b4ef3575fa8b89b8bd39b9457ad36722044 fixes it, but it depends on your case.
The error comes from `find` due to some limitations/bugs when used with `-L`.

**Reply by kenorb** [reply]
Fixes merged to `master`.
